include/string_utils.h

```diff
@@ -91,19 +91,9 @@ size_t safe_strlen(char (&str)[N]) noexcept
 	return strnlen(str, N - 1);
 }
 
-template <size_t N>
-bool starts_with(const char (&pfx)[N], const char *str) noexcept
-{
-	return (strncmp(pfx, str, N - 1) == 0);
-}
-
-template <size_t N>
-bool starts_with(const char (&pfx)[N], const std::string &str) noexcept
-{
-	return (strncmp(pfx, str.c_str(), N - 1) == 0);
-}
+bool starts_with(const std::string_view prefix, const std::string_view str) noexcept;
```

Comment:
While you're at it, could you please switch the order of arguments so it's consistent with `starts_with` and general conventions where the first arg is the receiver/main operand?

I've done this change in my capture PR already, btw.

Reply:
Will do!